lib/child-writer/writer-proxy.js

```diff
@@ -157,6 +158,13 @@ return isElevated().then((elevated) => {
       message: 'User did not grant permission.'
     }, () => {
       process.exit(EXIT_CODES.CANCELLED);
+    }).catch({
+      message: 'No polkit authentication agent found.'
+    }, () => {
+      throw errors.createUserError(
+        'No polkit authentication agent found',
+        'Please install a polkit authentication agent for your desktop environment of choice to continue'
```

Comment:
This error message is perhaps a bit obscure for the typical user that Etcher is aimed at? I wonder if we should maybe include a URL that the user could visit for more information?
(Or is it only likely to be 'hardcore' users that are running without a 'polkit authentication agent' anyway?)